.github/kokoro/continuous-test.cfg

```diff
@@ -0,0 +1,18 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+# Generated from .github/kokoro/kokoro-cfg.py
+# To regenerate run:
+# cd .github/kokoro/ && python3 kokoro-cfg.py
+build_file: "UHDM-continuous-test/.github/kokoro/test.sh"
```

Comment:
I'm pretty sure this file path is wrong?

Reply:
Fixed, I was looking at wrong commit in other repo.